components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/soaptorest/SequenceGenerator.java

```diff
@@ -0,0 +1,408 @@
+                        JSONObject resource = (JSONObject) entry.getValue();
+
+                        Set methods = resource.keySet();
+                        for (Object key1 : methods) {
+                            String method = (String) key1;
+
+                            List<JSONObject> mappingList = SequenceUtils
+                                    .getResourceParametersFromSwagger(apiJSON, resource, method);
+                            String inSequence = generateApiInSequence(mappingList, soapOperations, resourcePath,
+                                    method);
+                            String outSequence = generateApiOutSequence();
+                            if (log.isDebugEnabled()) {
+                                log.debug("Generated api in sequence for " + resource + " is: " + inSequence);
+                                log.debug("Generated api out sequence for " + resource + " is: " + outSequence);
+                            }
+                            String resourceInPath = APIConstants.API_LOCATION + RegistryConstants.PATH_SEPARATOR +
+                                    provider + RegistryConstants.PATH_SEPARATOR + name
+                                    + RegistryConstants.PATH_SEPARATOR + version + RegistryConstants.PATH_SEPARATOR
+                                    + SOAPToRESTConstants.SEQUENCE_GEN.SOAP_TO_REST_IN_RESOURCE + resourcePath
+                                    + SOAPToRESTConstants.SEQUENCE_GEN.RESOURCE_METHOD_SEPERATOR + method
+                                    + SOAPToRESTConstants.SEQUENCE_GEN.XML_FILE_EXTENSION;
+                            String resourceOutPath = APIConstants.API_LOCATION + RegistryConstants.PATH_SEPARATOR +
+                                    provider + RegistryConstants.PATH_SEPARATOR + name
+                                    + RegistryConstants.PATH_SEPARATOR + version + RegistryConstants.PATH_SEPARATOR
+                                    + SOAPToRESTConstants.SEQUENCE_GEN.SOAP_TO_REST_OUT_RESOURCE + resourcePath
+                                    + SOAPToRESTConstants.SEQUENCE_GEN.RESOURCE_METHOD_SEPERATOR + method
+                                    + SOAPToRESTConstants.SEQUENCE_GEN.XML_FILE_EXTENSION;
+                            SequenceUtils.saveRestToSoapConvertedSequence(registry, inSequence, method, resourceInPath);
+                            SequenceUtils
+                                    .saveRestToSoapConvertedSequence(registry, outSequence, method, resourceOutPath);
+                        }
+                    }
+                }
+            } catch (RegistryException e) {
+                handleException("Error when create registry instance ", e);
+            } catch (UserStoreException e) {
+                handleException("Error while reading tenant information ", e);
+            } catch (ParseException e) {
+                handleException("Error while parsing soap operations json content", e);
```

Comment:
any reasong this sentence doenst has a space at the end. If it's not needed remove from other places as well